src/tetris/brick/brickType.ts

```diff
@@ -0,0 +1,5 @@
+enum BrickType {
+    'I', 'O', 'J', 'L', 'T', 'S', 'Z'
```

Comment:
Usually, enum initialization should be done without quotes

Reply:
done